### DISCUSSION TITLE
PDF 2.0 reference (ISO 32000-2:2020)

### DISCUSSION THREAD

**Discussion by andersonhc** [py-pdf/fpdf2#842] [General]
I have been chasing this document for a while - even considered buying the ISO - and I just learnt the PDF Association is sponsoring the ISO so everyone can download for free.

https://www.pdfa.org/sponsored-standards

Now that I have the ISO my next step is starting a PR to implement AES-256 encryption on fpdf2 :)

**Comment by Lucas-C** [comment]
Nice! Thank you @andersonhc

For reference, the PDF 1.8 spec is made available by Adobe there:
https://opensource.adobe.com/dc-acrobat-sdk-docs/standards/pdfstandards/pdf/PDF32000_2008.pdf

**Comment by leoluke** [comment]
PDF 2.0 in Adobe 
PDF 2.0 (ISO 32000-2 / 2017)
https://developer.adobe.com/document-services/docs/assets/5b15559b96303194340b99820d3a70fa/PDF_ISO_32000-2.pdf
